### DISCUSSION TITLE
RuntimeError: Compute homography failed. No matched keypoints found.

### DISCUSSION THREAD

**Discussion by BillLeeMY** [kornia/kornia#2156] [Questions]
Hi guys, I am new to kornia library and I have a question regarding to the ImageStitcher.
I try with my device with the example code provided in https://kornia.readthedocs.io/en/latest/applications/image_stitching.html, I encounter an error.
    
raise RuntimeError("Compute homography failed. No matched keypoints found.")
RuntimeError: Compute homography failed. No matched keypoints found.

Then, I tried to print the output from image_stitching.py , I found out that the correspondences has no value inside.
This is the correspondences
{'keypoints0': tensor([], size=(0, 2)), 'keypoints1': tensor([], size=(0, 2)), 'confidence': tensor([]), 'batch_indexes': tensor([], dtype=torch.int64)}

Does anyone know why is this happening?

**Comment by edgarriba** [comment]
Hi, can you share a with an example reproducing this error ? ‘cc @ducha-aiki
